app/src/policy_engine.h

```diff
@@ -151,10 +278,13 @@ namespace scitt
       }
 
       auto phdr_val = protected_header_to_js_val(interpreter, phdr);
+      auto uhdr_val = unprotected_header_to_js_val(interpreter, uhdr);
+      auto payload_val = interpreter.new_array_buffer_copy(payload);
+      auto details_val = verified_details_to_js_val(interpreter, details);
 
       const auto result = interpreter.call_with_rt_options(
         apply_func,
-        {phdr_val},
+        {phdr_val, uhdr_val, payload_val, details_val},
```

Comment:
Because we are now passing the entire payload to the JS interpreter, do we need to increase the max_stack_bytes limit set below? Since the entire payload can go up to 1MB, I suppose the stack should be higher than that to accommodate for large parameters.